RELEASE.md

```diff
@@ -50,8 +50,8 @@ Would you like to contribute to the libp2p project and don't know how? Well, the
 - Check the issues with the `help wanted` label in the [libp2p repo](https://github.com/libp2p/js-libp2p/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
 - Join an IPFS All Hands, introduce yourself and let us know where you would like to contribute - https://github.com/ipfs/team-mgmt#all-hands-call
 - Hack with IPFS and show us what you made! The All Hands call is also the perfect venue for demos, join in and show us what you built
-- Join the discussion at http://discuss.ipfs.io/ and help users finding their answers.
-- Join the [⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 ](https://github.com/ipfs/team-mgmt/issues/650) and be part of the Sprint action!
+- Join the discussion at http://discuss.libp2p.io/ and help users finding their answers.
+- Join the [⚡️IPFS Core Implementations Weekly Sync 🙌🏽 ](https://github.com/ipfs/team-mgmt/issues/992) and be part of the Sprint action!
```

Comment:
should we change this for the libp2p sync?

Reply:
Yeah, I think that's a better move 👍